smithy-aws-iam-traits/src/test/resources/software/amazon/smithy/aws/iam/traits/errorfiles/define-conditionkeys-required-booleancheck.smithy

```diff
@@ -0,0 +1,19 @@
+$version: "2"
```

Comment:
I don't think we need this test, as it's not validating behavior specific to the trait - it's validation for model loading and parsing.

Reply:
removed